promtimer/annotations.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2020 Couchbase, Inc All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from os import path
+import time
+import json
+from dateutil import parser as dateparser
+import urllib.request
+
+ANNOTS_URL = 'http://localhost:13300/api/annotations'
+ANNOTS_HEADERS = {
+    'Accept': 'application/json',
+    'Content-Type': 'application/json',
+}
+ANNOTS_EVENTS_START = {
+    'rebalance_start': 'rebalance',
+    'failover_start': 'failover',
+}
+ANNOTS_EVENTS_END = {
+    'rebalance_finish': 'rebalance',
+    'failover_finish': 'failover',
+}
+ANNOTS_EVENT_TAGS = {
+    # 'dataset_created': 'success',
+    # 'dataset_dropped': 'warning',
+
+    'analytics_index_created': 'success',
+    'analytics_index_dropped': 'warning',
+
+    # 'task_finished': 'success',
+    # 'task_started': 'info',
+
+    'backup_removed': 'warning',
+    'backup_paused': 'warning',
+    'backup_resumed': 'info',
+    'backup_plan_created': 'success',
+    'backup_plan_deleted': 'warning',
+    'backup_repo_created': 'success',
+    'backup_repo_deleted': 'warning',
+    'backup_repo_imported': 'success',
+    'backup_repo_archived': 'success',
+
+    'rebalance_start': 'info',
+    'rebalance_finish': 'info',
+
+    'failover_start': 'warning',
+    'failover_end': 'warning',
+
+    'node_joined': 'success',
+    'node_went_down': 'warning',
+
+    # 'eventing_function_deployed': 'success',
```

Comment:
I would either leave these in or just remove them. It's not clear what leaving them in commented out achieves.